src/main/java/com/zufar/icedlatte/email/api/EmailTokenSender.java

```diff
@@ -14,22 +12,10 @@ public class EmailTokenSender {
 
     private final AuthTokenEmailConfirmation emailConfirmation;
     private final TokenManager tokenManager;
-    private final TokenCache tokenCache;
 
-    @Value("${temporary-cache.get-token-enabled}")
-    private boolean getTokenEnabled = false;
 
     public void sendEmailVerificationCode(final UserRegistrationRequest request) {
         String token = tokenManager.generateToken(request);
-        emailConfirmation.sendTemporaryCode(request.email(), token);
-    }
-
-    // for test purposes only
-    public String getEmailVerificationCode(final UserRegistrationRequest request) {
-        if (getTokenEnabled) {
-            return tokenCache.getToken(request);
-        } else {
-            return null;
-        }
```

Comment:
I'm afraid this method was requested by QA , we should probably discuss it with Tanya